test/e2e/parallel/proxy_test.go

```diff
@@ -232,14 +233,14 @@ func TestProxyFlow(t *testing.T) {
 				}
 
 				t.Run("use proxy to update a HAS Application CR in the user appstudio namespace via proxy API", func(t *testing.T) {
-					// Update application
+					// Get application name
 					applicationName := user.getApplicationName(0)
-					// Get application
-					proxyApp := user.getApplication(t, proxyCl, applicationName)
 					// Update DisplayName
 					changedDisplayName := fmt.Sprintf("Proxy test for user %s - updated application", tenantNsName(user.compliantUsername))
-					proxyApp.Spec.DisplayName = changedDisplayName
-					err := proxyCl.Update(context.TODO(), proxyApp)
+					// Update application
+					proxyApp, err := awaitilities.Host().UpdateApplication(t, proxyCl, applicationName, tenantNsName(user.compliantUsername), func(app *appstudiov1.Application) {
+						app.Spec.DisplayName = changedDisplayName
+					})
 					require.NoError(t, err)
```

Comment:
Not related to this PR, but I think that we can drop the "AppStudio API"-specific test cases. It's completely fine to test modification on standard k8s resources (eg. ConfigMap), no need to maintain the other API

Reply:
Apart from that, there is actually no Application controller running that would watch/modify these resources. This means that we don't expect any conflicts for this resource and the update through the proxy should work fine without having the loop.

Reply:
Thank you so much! I will remove the loop here. And take care of the "AppStudio API"-specific test cases in a different PR